dist/generator-vulcanjs/lib/ui-text.js

```diff
@@ -22,6 +23,7 @@ const descriptions = {
 
 const messages = {
   appName: 'App name',
+  doShallowClone: 'Use fast (<5s) or complete (>60s) git clone? See unshallow action for more "fast" details.',
```

Comment:
Again, I think the formulation is not optimal, but keeps things simple. 

As discussed on slack, this is the hot fix version, the road ahead will avoid having to clone the full repo each time.